Readme.md

```diff
@@ -1,24 +1,20 @@
-# WebMemex/memextension
+# WorldBrain's WebMemex
```

Comment:
So now you also want to put your project name in the WebMemex title? It's great you want to use and contribute to this project, but this may be a bit too much. ;)